### PR TITLE
Contact-Section

### PR DESCRIPTION
**Pull request by Abhiram716**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "gatsby-transformer-remark": "^6.11.0",
     "gatsby-transformer-sharp": "^5.11.0",
     "react": "^18.2.0",
+    "react-copy-to-clipboard": "^5.1.0",
```
Comment (aadilmughal786) [inline package.json:31]: We have already native API for this, is this really required

https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API

File: src/components/Contact/Socials.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { IconButton, Tooltip, useTheme } from "@mui/material";
+import React, { useState } from "react";
+import { BsGithub, BsLinkedin, BsStackOverflow } from "react-icons/bs";
+
+const Socials = () => {
+  const theme = useTheme();
+  const socialsData = [
+    {
+      icon: <BsGithub color={theme.palette.primary.contrastText} />,
```
Comment (aadilmughal786) [inline src/components/Contact/Socials.jsx:9]: const {contrastText} = theme.palette.primary

would be better

File: src/components/Contact/Socials.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    {
+      icon: <BsStackOverflow color={theme.palette.primary.contrastText} />,
+      title: "Stackoverflow",
+      redirect: "https://stackoverflow.com/users/15342341/abhi-ram",
+    },
+  ];
+
+  return (
+    <>
+      {socialsData.map((social, index) => (
```
Comment (aadilmughal786) [inline src/components/Contact/Socials.jsx:27]: In this case index is ok as a key, but you can use title here as key

**Review by aadilmughal786 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Abhiram716) [reply package.json:31]: @aadilmughal786 ,

- I had no idea there was a native API for Clipboard when I was looking into this feature, and when I explored this dependency, I believed the simplicity of usage was great for me, so I didn't dig more.
- After you pointed out that there is a native API for the clipboard I saw the difference between both I found out that there is not a lot react-copy-to-clipborad does. simply, dependency is not required (except for adding a dependency for my project). Now switching to native API

Thanks for pointing that out.

**Review by aadilmughal786 — APPROVED** [review]

**Pull request merged**